addons/mail/static/src/views/activity/activity_model.js

```diff
@@ -17,3 +23,5 @@ export class ActivityModel extends RelationalModel {
         });
     }
 }
+
+ActivityModel.DynamicRecordList = ActivityDynamicRecordList;
```

Comment:
Can you declare using `static DynamicRecordList = ActivityDynamicRecorldList` instead? So we are consistent with other Activity related classes